### PR TITLE
Modified README.md to be more noob friendly + added info found only on the screengrab github doc

### PR DESCRIPTION
**Pull request by sohaeb**
As a new Android developer, the original README.md is not consistent with the main screengrab github page. Moreover,  the organization of this file is not really good.

To solve this. I did 3 things:

1. Copied the nice information found in the README.md file of the screengrab's github page and placed them inside this doc. Now, there is no need for someone to read 2 different docs on 2 different locations to learn about screengrab.

2. I  added a new youtube link to help newbies get into espresso UI testing. Watching  this video  is more than enough and can save you couple of hours of reading.

3. I moved the paragraph "Improved screenshot capture with UI Automator" to be under "Espresso" paragraph this way, users will know about the new approach before actually trying to upload the screenshots.

### CODE REVIEW

File: docs/getting-started/android/screenshots.md [hunk 1]
```diff
@@ -25,7 +25,9 @@ After _screengrab_ completes, it will show you summary of the screenshots you ca
...
+_screengrab_ uses the capabilities of Android's built-in [Instrumented tests](https://developer.android.com/training/testing/unit-testing/instrumented-unit-tests.html) combined with [Espresso](https://developer.android.com/topic/libraries/testing-support-library/index.html#Espresso) to drive interactions with your app. 
+
+To get started with writing UI Tests with Espresso, [Create UI tests with Espresso Test Recorder](https://www.youtube.com/watch?v=JRkDVvB106k) video gives a quick intro to Espresso Recording feature for writing UI tests. For more details regrading UI testing, refer to the Anroid guide: [Testing UI for a Single App](http://developer.android.com/training/testing/ui-testing/espresso-testing.html).
```
Comment (taquitos) [inline docs/getting-started/android/screenshots.md]: typo: "Anroid guide"

Comment (taquitos) [inline docs/getting-started/android/screenshots.md]: add "the" before `[Create UI tests with Espresso Test Recorder]`

Comment (taquitos) [inline docs/getting-started/android/screenshots.md]: After `video gives a quick into to`
add `the` before `Espresso Recording feature`

**Comment by taquitos** [comment]
🐑

**Comment by KrauseFx** [comment]
Oh yeah 🎉

**Comment by KrauseFx** [comment]
🌮

**Pull request merged**